### PR TITLE
MSPSDS-285: Rename investigations to cases in the UI

### PR DESCRIPTION
**Pull request by broder**

### CODE REVIEW

File: app/views/activities/show.html.erb [hunk 1]
```diff
@@ -1,53 +0,0 @@
-<% content_for :page_title, "View activity - Market Surveillance & Product Safety" %>
```
Comment (AnthonyPengelly) [inline app/views/activities/show.html.erb]: Did I not delete this already? Oops

File: app/views/investigations/edit.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <% content_for :page_title, "Edit investigation - Market Surveillance & Product Safety" %>
```
Comment (AnthonyPengelly) [inline app/views/investigations/edit.html.erb]: I think you've missed a few of these page titles

Comment (broder) [reply app/views/investigations/edit.html.erb]: Good spot, thanks!

**Comment by tom-bell-softwire** [comment]
Thanks for this! I think there's another reference buried in `images/index.html.erb`:

```ruby
<%= link_to "Back to #{@parent.class.name.downcase}" ...
```

Think the only way to handle it is with an `if-else` for Investigation, unfortunately...

**Pull request merged**